src/frontend/web_application/src/styles/object/o-avatar.scss

```diff
@@ -2,7 +2,8 @@ $m-avatar__size: (
   small:  2rem,
   medium: 3rem,
   large:  4rem,
-  xlarge: 6rem
+  xlarge: 6rem,
+  xxlarge: 8rem
```

Comment:
I agree with you. 

May be the problem is the notation: `small, xxlarge ... ` are not good at the end if we have more than 3 sizes. 

This can be a good case for http://cssinjs.org on the other side extending styles can be a bad thing since allows inconsistent designs.